qa/tasks/cephfs/test_exports.py

```diff
@@ -25,6 +25,7 @@ def _wait_subtrees(self, status, rank, test):
     def test_export_pin(self):
         self.fs.set_allow_multimds(True)
         self.fs.set_max_mds(2)
+        self.fs.wait_for_daemons()
```

Comment:
thanks for clarification. I tired 'ceph health' command, didn't read are_daemons_healthy() code.